sdk/schemaregistry/schema-registry-json/test/public/errors.spec.ts

```diff
@@ -165,6 +165,9 @@ describe("Error scenarios", function () {
       );
     });
     it("schema with invalid ID", async function () {
+      if (!isLiveMode()) {
```

Comment:
nitpick: we can also update the mock registry to throw the same error but maybe it is not worth the trouble.